docs/user_manual/calculations.md

```diff
@@ -554,12 +563,102 @@ The assumptions used for calculations in power-grid-model are aligned to the one
+        - If the $U_{\text{control}} < U_{\text{set}} - \frac{U_{\text{band}}}{2}$:
+          - The ratio between the voltage on the tap side and the voltage on the control side is too high.
+          - To decrease this ratio, the tap ratio must be decreased.
+          - Therefore, the tap position of the regulated transformer is decreased if it satisfies the bounds set by `tap_min` and `tap_max`.
+        - If, however, the $U_{\text{control}} > U_{\text{set}} + \frac{U_{\text{band}}}{2}$:
+          - The ratio between the voltage on the tap side and the voltage on the control side is too low.
+          - To increase this ratio, the tap ratio must be increase.
+          - Therefore, the tap position of the regulated transformer is increased if it satisfies the bounds set by `tap_min` and `tap_max`.
+        - If the tap position of this transformer did not change, the transformer is considered regulated.
+      - If not all transformers within this rank are regulated:
+        - A better combination of tap positions may have been found.
+        - Step out of the loop and go to the next iteration step.
+  - Exploit the neighbourhood of all transformers (see {hoverxreftooltip}`user_manual/calculations:Initialization and exploitation of regulated transformers`)
+    - Re-run the iteration in the above if any of the tap positions changed by the exploitation.
+
+The exploitation of the neighbourhood ensures that the actual optimum is not accidentally missed due to feedback mechanisms in the grid.
+
+```{note}
+For iterative [power flow calculation methods](#power-flow-algorithms), the initial state may not converge.
+If the iterative process failed to converge, the optimization procedure is executed twice.
+First, the {py:class}`linear <power_grid_model.enum.CalculationMethod.linear>` calculation method is used to find an approximate solution.
+The optimization procedure is then run a second time to find the actual optimum with the user-specified calculation method.
+```
+
+```{note}
+The control logic assumes that the change in voltage level caused by changing a transformer is larger than the change caused by adjacent transformers.
+This is assumption is reflected in the requirements mentioned in {hoverxreftooltip}`user_manual/components:Transformer Tap Regulator`.
+```
+
+```{note}
+The control logic assumes that the (compensated) control voltage decreases when the tap position increases.
+If the line drop compensation impedance is high, and the control side has generator-like behavior, then this assumption does not hold, and the calculation may diverge.
+Hence, this assumption is reflected in the requirements mentioned in {hoverxreftooltip}`user_manual/components:Line drop compensation`.
+```
+
+##### Initialization and exploitation of regulated transformers
+
+Regulated transformers are initialized and exploited depending on the optimization strategy.
```

Comment:
In the context of all written text about `exploitation`, I think more information can be added here or below: 

Above the table,
Internally, the control algorithm sets initial tap position as following so we end up with a regulated tap_pos corresponding to the following conditions:

In the description section
`Find best possible tap position that gives highest control side voltage within the u_band.`
`Find best possible tap position that gives lowest control side voltage within the u_band.`

Tap side voltage seems less relevant i suppose.